mix.exs

```diff
@@ -47,7 +47,7 @@ defmodule ExDoc.Mixfile do
       {:makeup_html, ">= 0.1.0", optional: true},
       {:jason, "~> 1.2", only: :test},
       {:floki, "~> 0.0", only: :test},
```

Comment:
`easyhtml` depends on `floki`, it was likely used before, that's why both are listed. Looks like there is one direct usage in `erlang_test.exs`, which we should be able to replace with `lazy_html` as well :)

```suggestion
```